service/history/api/get_workflow_util.go

```diff
@@ -209,7 +209,7 @@ func MutableStateToGetResponse(
 		CurrentBranchToken:                    currentBranchToken,
 		WorkflowState:                         workflowState,
 		WorkflowStatus:                        workflowStatus,
-		IsStickyTaskQueueEnabled:              mutableState.IsStickyTaskQueueEnabled(),
+		IsStickyTaskQueueEnabled:              mutableState.TaskQueue().Kind == enumspb.TASK_QUEUE_KIND_STICKY,
```

Comment:
This is slightly longer but also more verbose.

Reply:
I think the intent behind "more verbose" is actually "more intention revealing," which is true. IsStickyTaskQueueEnabled could mean a number of different things. If we want to be intention-revealing and also encapsulate this comparison, I'd suggest a free function: `workflow.IsSticky(*taskqueuepb.TaskQueue)`. Or if you're feeling fancy, `workflow.IsSticky(interface { GetKind() taskqueuepb.TaskQueueKind })`

Reply:
I am bringing it back under different name: `IsStickyTaskQueueSet()`.